assets/i18n/hi.json

```diff
@@ -0,0 +1,276 @@
+  "I18N_FOOTER_BROWSE_LIBRARY": "पुस्तकालय ब्राउज़ करे",
+  "I18N_FOOTER_CONTACT_US": "संपर्क करें",
+  "I18N_FOOTER_CONTRIBUTE_ALL_CAPS": "योगदान",
+  "I18N_FOOTER_CREDITS": "श्रेय",
+  "I18N_FOOTER_DONATE": "दान करे",
+  "I18N_FOOTER_FORUM": "मंच",
+  "I18N_FOOTER_GET_INVOLVED": "शामिल होईये",
+  "I18N_FOOTER_OPPIA_FOUNDATION": "Oppia संस्थान",
+  "I18N_FOOTER_PARTICIPATION_PLAYBOOK": "भागीदारी प्लेबुक",
+  "I18N_FOOTER_PRIVACY_POLICY": "गोपनीयता",
+  "I18N_FOOTER_TEACH": "Oppia के साथ  सिखाये",
+  "I18N_FOOTER_TEACH_LEARN_ALL_CAPS": "सिखाये/सीखे",
+  "I18N_FOOTER_TERMS_OF_SERVICE": "शर्तें",
+  "I18N_FORMS_TYPE_NUMBER": "एक नंबर टाइप करे",
+  "I18N_FORMS_TYPE_NUMBER_AT_LEAST": "एक संख्या लिखे जो कम से कम  <[minValue]> हो।",
+  "I18N_FORMS_TYPE_NUMBER_AT_MOST": "एक संख्या लिखे जो ज्यादा से ज्यादा  <[maxValue]> हो।",
+  "I18N_FORMS_TYPE_NUMBER_INVALID_DECIMAL": "एक वैध दशमलव संख्या लिखे।",
+  "I18N_INTERACTIONS_GRAPH_ADD_EDGE": "धार जोड़े ",
+  "I18N_INTERACTIONS_GRAPH_ADD_NODE": "नोड जोड़े",
+  "I18N_INTERACTIONS_GRAPH_DELETE": "मिटाये",
+  "I18N_INTERACTIONS_GRAPH_ERROR_INVALID": "अवैध ग्राफ!",
+  "I18N_INTERACTIONS_GRAPH_MOVE": "हिलाये",
+  "I18N_INTERACTIONS_GRAPH_RESET_BUTTON": "रीसेट",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_EDGE": "<[edges]> धार",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_EDGES": "<[edges]> धार",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_VERTEX": "तथा <[vertices]> कोने",
+  "I18N_INTERACTIONS_GRAPH_RESPONSE_VERTICES": "तथा <[vertices]> कोने",
+  "I18N_INTERACTIONS_GRAPH_SUBMIT_BUTTON": "जमा करें",
+  "I18N_INTERACTIONS_GRAPH_UPDATE_LABEL": "लेबल अपडेट करे",
+  "I18N_INTERACTIONS_GRAPH_UPDATE_WEIGHT": "वजन अपडेट करे",
+  "I18N_INTERACTIONS_IMAGE_CLICK_SELECT": "[प्रदर्शित करने के लिए एक छवि का चयन करे]",
+  "I18N_INTERACTIONS_ITEM_SELECTION_NOT_ENOUGH": "{minChoiceNumber, plural, one{कृपया कम से कम एक विकल्प का चयन करे। } other{कम से कम # का चयन करें ।}}",
+  "I18N_INTERACTIONS_ITEM_SELECTION_PREVENT_MORE": "{maxAllowableSelectionCount, plural, one{ज्यादा से ज्यादा 1 पसंद को चुना जा सकता है ।} other{ज्यादा से ज्यादा # पसंद को चुना जा सकता है ।}}",
+  "I18N_INTERACTIONS_ITEM_SELECTION_SUBMIT": "जमा करें",
+  "I18N_INTERACTIONS_LOGIC_PROOF_CORRECT_ANSWERS": "सही: <[answer]>",
+  "I18N_INTERACTIONS_LOGIC_PROOF_INCORRECT_ANSWERS": "ग़लत: <[answer]>",
+  "I18N_INTERACTIONS_LOGIC_PROOF_POSSIBLE_LINES": "संभव लाइनों",
+  "I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_ASSUMPTIONS": "यह सोचते हुए  <[assumptions]>; साबित करे <[target]>.",
+  "I18N_INTERACTIONS_LOGIC_PROOF_QUESTION_STR_NO_ASSUMPTION": "साबित करे  <[target]>.",
+  "I18N_INTERACTIONS_LOGIC_PROOF_SUBMIT": "जमा करें",
+  "I18N_INTERACTIONS_MUSIC_CLEAR": "मिटाये",
+  "I18N_INTERACTIONS_MUSIC_PLAY": "चलाएँ",
+  "I18N_INTERACTIONS_MUSIC_PLAY_SEQUENCE": "लक्ष्य अनुक्रम चलाएँ",
+  "I18N_INTERACTIONS_MUSIC_SUBMIT": "जमा करें",
+  "I18N_INTERACTIONS_SET_INPUT_ADD_ITEM": "जोडें",
+  "I18N_INTERACTIONS_SET_INPUT_DUPLICATES_ERROR": "ओह, यह लग रहा है की आपके सेट में कुछ सामान आइटम है !",
```

Comment:
How about "ओह, यह लग रहा है की आपके सेट में कुछ समान वस्तुएं है " ?
I think item should be replaced with something more appropriate in that context. Any thoughts ?
Also replace "सामान" with "समान" (typo).